test/skimNtuple_HHbtag.cpp

```diff
@@ -4029,15 +4035,6 @@ int main (int argc, char** argv)
 		  vector <TLorentzVector> tlv_secondBjet_raw_jetup(N_jecSources, tlv_secondBjet_raw);
 		  vector <TLorentzVector> tlv_secondBjet_raw_jetdown(N_jecSources,tlv_secondBjet_raw);
 
-		  //        if (DEBUG)
-		  //        {
-		  //            cout << "-------- JET JEC -------" << endl;
-		  //            cout << "jet1 UP: " << tlv_firstBjet_raw_jetup.Pt() <<endl;
-		  //            cout << "jet1 DW: " << tlv_firstBjet_raw_jetdown.Pt()<<endl;
-		  //            cout << "jet2 UP: " << tlv_secondBjet_raw_jetup.Pt()<<endl;
-		  //            cout << "jet2 DW: " << tlv_secondBjet_raw_jetdown.Pt()<<endl;
-		  //        }
-
 		  theSmallTree.m_bjet1_pt_raw = tlv_firstBjet_raw.Pt();
```

Comment:
this (and other `m_bjet*` variables should probably be filled outside of the if statement too, to get consistent dummy values for events with less than 2 b-jet candidates (as discussed with @kramerto)

Reply:
I've defined appropriate dummy values for ```eta``` and ```phi``` instead.